### PR TITLE
Fixes #154 Make student explorer easier for development

### LINKED ISSUE #154: Make student explorer reloadable for easier development
@ssciolla was saying he'd like SE to be reloadable for easier development. This should be something we implement.

### PR DESCRIPTION
**Pull request by jonespm**

### CODE REVIEW

**Comment by jonespm** [comment]
I've been using this locally but would like to have one person approve. This really only changes docker-compose so will just affect localhost. 

The start.sh changes were typos that also were only for development.

**Comment by mfldavidson** [comment]
@ssciolla since this was your idea could you review?

**Comment by ssciolla** [comment]
Yup, sorry for the delay, I'll review this morning.

**Comment by jonespm** [comment]
Looks like I missed an if statement in the start script, test again. I was testing with PTVSD_ENABLE=True so didn't see this.

File: start.sh [hunk 1]
```diff
@@ -33,9 +39,9 @@ if [ -z "${IS_CRON_POD}" ]; then
         # Currently ptvsd doesn't work with gunicorn
         # https://github.com/Microsoft/vscode-python/issues/2138
         echo Starting Runserver for development
-        export PYTHONPATH="/code:$PYTHONPATH"
+        export PYTHONPATH="/usr/src/app:$PYTHONPATH"
```
Comment (ssciolla) [inline start.sh]: It's odd this never caused a problem before...

**Comment by jonespm** [comment]
So the specifics are

The first line of the volumes on the Dockerfile adds the current directory to be put in place of the directory in /usr/src/app in the container. This is where Dockerfile puts the application. 

However there are some directories that have data that isn't present on the local host (static compiled files and local configuration files) the other 3 lines in the volumes say to not remap those files and leave those as is on the container.

The rest of it is just telling Gunicorn to reload if there's changes (With the --reload) parameter. If you're using django-admin runserver (currently still needed for ptvsd) it automatically reloads by default but I added nothreading as that helps debugging a little and should have been in there in the first place.

**Pull request merged**